docs/guides/high-availability/examples/generated/keycloak-ispn.yaml

```diff
@@ -486,8 +486,10 @@ spec:
       secret:
         name: remote-store-secret
         key: password
+    - name: cache-remote-backup-sites
```

Comment:
> I don't see where we suggest to use the availability-zone as the site name, where is it?

@ryanemerson here. In our guide, we use `site-1` and `site-2`, but it can be anything.